packages/babel-plugin-proposal-decorators/src/index.js

```diff
@@ -215,32 +215,29 @@ export default function() {
     inherits: syntaxDecorators,
 
     visitor: {
-      ExportDefaultDeclaration(path) {
```

Comment:
wonder if this will cause other plugin ordering issues?

We really should just move the class properties/decorators transforms into the same "plugin" and just turn them on in the individual pkgs

Reply:
It won't causes other problems that we don't already have (`Method has decorators, put the decorator plugin before the classes one.`)

Reply:
Really? If they are implemented in the same plugin wouldn't the issue be gone?